### PR TITLE
Changed airlocks to airlock helpers on NSS, Kerberos, NSS Cyberiad, NSS Cerebron

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Changed the old airlocks to airlock helpers

## Why It's Good For The Game
 Helps do some stuff, and when the map first boots the button spawn instead just being there

## Testing
 Spawned as explorer and made sure the buttons worked and they were in the right spot

## Changelog
:cl:
Tweak: changed some of the layouts of the airlocks to helpers on NSS, Kerberos, NSS Cyberiad, NSS Cerebron
/:cl:

### CODE REVIEW

**Comment by DGamerL** [comment]
> Del: I removed all the old airlocks on NSS, Kerberos, NSS Cyberiad, NSS Cerebron
Add: I replaced them with airlock helpers on NSS, Kerberos, NSS Cyberiad, NSS Cerebron

The changelog is only for player-facing stuff, so only tell things that players will need to know about, something like
`tweak: changed some layouts of ... etc.`

**Comment by Fluffyzack21** [comment]
I merged the fix with the merge master, Opps

File: .vscode/settings.json [hunk 1]
```diff
@@ -31,5 +31,6 @@
 	"[scss]": {
 		"editor.rulers": [80]
 	},
+	"GitHooks.hooksDirectory": "c:\\Users\\User1\\OneDrive\\Documents\\GitHub\\Paradise\\.git\\hooks",
```
Comment (warriorstar-orion) [inline .vscode/settings.json:34]: Revert this

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]
Sorry I missed this, looking at it now. You have a CI failure on Delta:
```
FAIL /datum/map_per_tile_test/pipe_vent_checker 0.1s
	84,137,2: pipe on same tile as vent or scrubber
```

**Comment by warriorstar-orion** [comment]
On Delta:

2024_02_04__17_00_40__paradise dme  delta dmm  - StrongDMM


This is a docking port and shouldn't use the airlock helper. It also shouldn't use the air canister so if you want to remove that as well, that would be good.

2024_02_04__17_03_55__paradise dme  delta dmm  - StrongDMM

This should almost certainly be a new airlock helper type that's gated on security access, not engineering.

Mentioned on Discord, but the CI error is due to this:

2024_02_04__17_08_45__paradise dme  delta dmm  - StrongDMM

You'll have to reroute the atmos pipes leading to minisat because the airlock helper places a vent over them, which isn't permitted.

File: .vscode/settings.json (hunk 1 above)
Comment (Fluffyzack21) [reply .vscode/settings.json]: fixed

**Comment by warriorstar-orion** [comment]
On delta:
2024_02_04__23_22_11__paradise dme  delta dmm  - StrongDMM

The airlock helper pipes aren't connected to the canister. I would also prefer you make a new subtype for security access helpers rather than varediting the existing spawner.

---

2024_02_04__23_27_47__paradise dme  delta dmm  - StrongDMM

This outlet pipe south of virology was changed.

---

2024_02_04__23_29_49__paradise dme  delta dmm  - StrongDMM

I thought you intended on using an airlock helper here? It looks like you rerouted the atmos pipes but put the normal external airlocks back.

---

On Box:

2024_02_04__23_34_42__paradise dme  cyberiad dmm  - StrongDMM

Fore Port Solar access should be the engineering access subtype.

---

2024_02_04__23_38_53__paradise dme  cyberiad dmm  - StrongDMM

Make an engineering access subtype for this helper near perma.

---

2024_02_04__23_42_46__paradise dme  cyberiad dmm  - StrongDMM

The helper near the AI sat should be engineering access.

**Comment by Fluffyzack21** [comment]
> On delta: 2024_02_04__23_22_11__paradise dme delta dmm - StrongDMM
> 
> The airlock helper pipes aren't connected to the canister. I would also prefer you make a new subtype for security access helpers rather than varediting the existing spawner.
> 
> 2024_02_04__23_27_47__paradise dme delta dmm - StrongDMM
> 
> This outlet pipe south of virology was changed.
> 
> 2024_02_04__23_29_49__paradise dme delta dmm - StrongDMM
> 
> I thought you intended on using an airlock helper here? It looks like you rerouted the atmos pipes but put the normal external airlocks back.
> 
> On Box:
> 
> 2024_02_04__23_34_42__paradise dme cyberiad dmm - StrongDMM
> 
> Fore Port Solar access should be the engineering access subtype.
> 
> 2024_02_04__23_38_53__paradise dme cyberiad dmm - StrongDMM
> 
> Make an engineering access subtype for this helper near perma.
> 
> 2024_02_04__23_42_46__paradise dme cyberiad dmm - StrongDMM
> 
> The helper near the AI sat should be engineering access.

should be fixed

**Comment by warriorstar-orion** [comment]
On Box:

2024_02_05__13_08_59__paradise dme  cyberiad dmm  - StrongDMM

You deleted a random tile in aft solar maints

---

2024_02_05__13_31_41__paradise dme  cyberiad dmm  - StrongDMM

Please create an engineering-access subtype and use it here:

```c
/obj/effect/spawner/airlock/s_to_n/long/engineer
	req_access_txt = ACCESS_ENGINE_EQUIP
	door_name = "engineering external access"
```

---

2024_02_05__13_34_51__paradise dme  cyberiad dmm  - StrongDMM

Please create an engineering-access subtype and use it here:

```c
/obj/effect/spawner/airlock/s_to_n/long/square/engineer
	req_access_txt = ACCESS_ENGINE_EQUIP
	door_name = "engineering external access"
```

---

On Delta:

2024_02_05__13_43_48__Paradise Station 13

The southern inner tile here needs to be a wall, otherwise the airlock controller will be floating in mid-air.

**Review by S34NW — CHANGES_REQUESTED** [review]
image
There is an air vent under this wall in delta engineering transit tubes
_____
image
image

These pipe are straight and not angled in delta north and south-east solars

_________

image
You deleted something from cyberiad maints

**Comment by S34NW** [comment]
> > image There is an air vent under this wall in delta engineering transit tubes
> > image image
> > These pipe are straight and not angled in delta north and south-east solars
> > image You deleted something from cyberiad maints
> 
> fixed

image
That vent is still there on delta

**Comment by Fluffyzack21** [comment]
> > > image There is an air vent under this wall in delta engineering transit tubes
> > > image image
> > > These pipe are straight and not angled in delta north and south-east solars
> > > image You deleted something from cyberiad maints
> > 
> > 
> > fixed
> 
> image That vent is still there on delta

Well gone now

**Pull request merged**